### PR TITLE
Doc fixes + Blockchain explorers

### PR DESCRIPTION
**Pull request by lock9**
- ImportBlocks is an optional plugin;
- "RPC Disabled" is actually RPCSecurity;
- Added a new page regarding available neo-explorers

### CODE REVIEW

**Comment by Celia18305** [comment]
I will hold this PR until “RPCSecurity” is released.

**Comment by lock9** [comment]
Hello @Celia18305. I just fixed the name of the plugin, RPC is already there, I just changed the name:
http://docs.neo.org/en-us/node/plugin.html
Is RPC Disabled not the same as RPC Security?

**Comment by Celia18305** [comment]
@lock9 Yes RPC Disabled will be changed to RPC Security as far as I know in 2.9.1, however, v2.9.1 has not been released at https://github.com/neo-project/neo-plugins/releases. That's why I prefer to update the doc after the new plugin name is released.

File: en-us/network/local-chain.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  - Block explorer \(Neoscan\);
+  - Notification server \(Neo-python\);
+  - Development tools \(Neo-python\);
+  - A wallet with GAS;
+  - Developer faucet (to obtain extra GAS);
+
+### Simple setup
+Having docker and docker-compose installed:
+  - Clone the repository:  https://github.com/CityOfZion/neo-local
+  - Run ```./neo-local/docker-compose up ``` from terminal or powershell
```
Comment (nunojusto) [inline en-us/network/local-chain.md]: This command is wrong.
You need to change the folder to be inside neo-local. And then Run ```docker-compose up```

Comment (Celia18305) [reply en-us/network/local-chain.md]: I made some changes. Can you please check is that right?

File: en-us/network/local-chain.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  - Notification server \(Neo-python\);
+  - Development tools \(Neo-python\);
+  - A wallet with GAS;
+  - Developer faucet (to obtain extra GAS);
+
+### Simple setup
+Having docker and docker-compose installed:
+  1. Clone the repository:  https://github.com/CityOfZion/neo-local
+  2. From terminal or powershell change to the newly cloned directory on your machine: ```cd ./neo-local ```. 
+  3. Run ```./docker-compose up ```.
```
[the remark this answers is not in the document]
Comment (Celia18305) [reply en-us/network/local-chain.md]: Fixed. Pls check again.

**Comment by lock9** [comment]
Looks ok to me! Thanks @nunojusto

**Pull request merged**